tlslite/tlsrecordlayer.py

```diff
@@ -147,6 +147,9 @@ def __init__(self, sock):
         #Limit the size of outgoing records to following size
         self.recordSize = 16384 # 2**14
 
+        # Indicator for heartbeat extension
+        self.heartbeat = False
```

Comment:
I was thinking of `heartbeatCanReceive` and `heartbeatCanSend`

remember that those values are used by both server and client, but the meaning is revered for `heartbeatClientMode` if you're server